frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.unit.spec.tsx

```diff
@@ -107,7 +107,25 @@ describe("TagEditorParam", () => {
   });
 
   describe("tag dimension", () => {
-    it("should default to string/= for a new string field filter", async () => {
+    it("should default to string/= for a new low cardinality string field filter", async () => {
+      const tag = createMockTemplateTag({
+        type: "dimension",
+        dimension: undefined,
+        "widget-type": undefined,
+      });
+      const { setTemplateTag } = setup({ tag });
+
+      userEvent.click(await screen.findByText("People"));
+      userEvent.click(await screen.findByText("Source"));
```

Comment:
nit: if you already waited for People, most likely Source is there already, so screen.getByText could work as well

Reply:
ignore it please